rules/diff_test.bzl

```diff
@@ -120,14 +170,19 @@ else
   echo >&2 "ERROR: could not find \"{file1}\" and \"{file2}\""
   exit 1
 fi
-if ! diff "$RF1" "$RF2"; then
-  echo >&2 "FAIL: files \"{file1}\" and \"{file2}\" differ. "{fail_msg}
+if ! diff {strip_trailing_cr}"$RF1" "$RF2"; then
+  MSG={fail_msg}
+  if [[ "${{MSG}}" == "" ]]; then
+      MSG="why? diff {strip_trailing_cr}"${{RF1}}" "${{RF2}}" | cat -v"
```

Comment:
Why is this message so different from the message before? Also use of RF1 is inconsistend with use of {file1} below.